client/my-sites/plans/navigation.js

```diff
@@ -5,6 +5,7 @@
 import PropTypes from 'prop-types';
```

Comment:
This file contains a React component, so IMHO it would be more suitable to keep the `.jsx` extension - some editors don't recognize JSX when the file format is .JS.

Reply:
I really dislike the `jsx` extension, I find it to be an annoyance around build tools.

That said, I don't think we'll wipe out jsx anytime soon and you have a pragmatic reason for wanting it. I'll change it.

Reply:
30a03e46f0